spec/requests/v1/interventions/stars/put_star_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'PUT /v1/interventions/:id/star', type: :request do
```

Comment:
I think "POST" would be better

Reply:
I have discussed this topic with @bartosz-kepka-htd, and in his opinion using PUT will be better, as PUT is idempotent, compared to POST, aka invoking this endpoint multiple times shouldn't result in multiple stars between the same user and intervention

Reply:
You can define a post in the same way as in "V1::UserSessionsController#create". POST indicates that you create a new record and "PUT" that you only modify. Looking at this I still believe the "POST" suits better

Reply:
PUT in rails works exactly as PATCH. As you can see on the attached screen by default rails connect PUT/PATH to the same method in the controller 
<img width="1115" alt="Screenshot 2023-08-24 at 06 48 25" src="https://github.com/Michigan-State-University/cias-api/assets/61981027/2ebb2048-2f5f-4b93-bd5b-27620fd18b7b">